bot.py

```diff
@@ -0,0 +1,65 @@
+from twitchio.ext import commands
+import asyncpg
+import openai
+import os
+import dotenv
+
+dotenv.load_dotenv()
+
+async def verse_generation(text):
+    openai.api_key = os.getenv("OPENAPI_KEY")
+
+    verse = openai.ChatCompletion.create(
+        model="gpt-3.5-turbo", 
+        messages=[{
+            "role": "user", 
+            "content": f"Create a verse that rhymes up to 3 lines using the following message: {text}"
```

Comment:
will chatgpt handle english instructions with portuguese text?
also, this is very prone to prompt injection